test/test.js

```diff
@@ -74,8 +81,26 @@ test('works when destructuring isnt available natively', t => {
 	t.is(result.y, 'b');
 });
 
-test('parse React fragments', t => {
-	t.notThrows(() => {
-		importJsx(fixturePath('react-fragment'));
-	});
+const diskCacheFile = `${diskCacheDirectory}/f478c8d5f1d242d6e659f8c40f33374c.js`;
+
+test('creates appropriate cache file on disk', t => {
+	t.false(fs.existsSync(diskCacheFile));
+
+	importJsx(fixturePath('for-cache'), {cache: false});
```

Comment:
If cache is turned off, why does it create a cache file?

Reply:
The "cache" option is for the existing in memory cache, which this PR doesn't change. I didn't add an option for the new disk cache. If you think it's a good idea, I could rename the existing "cache" option to "cacheMemory" and add a new option "cacheDisk" to enable the new disk cache.

Reply:
I think `cache` option should enable/disable both disk and transform cache.

Reply:
That makes sense. I just made that change - now the `cache` option affects both caches.